### PR TITLE
TK274

### PR DESCRIPTION
**Pull request by karuna835**
- Solved SIS file issue.

### CODE REVIEW

File: app/modules/user_enrollment/sis.php [hunk 1]
```diff
@@ -181,10 +181,10 @@ function sis_records_process($subs_id = false) {
 				$users_done[] = $enr["user_id"];
 
 			}
-			if(!empty($is_alumni) && $is_alumni[0]['is_prev_alumni'])
+			/*if(!empty($is_alumni) && $is_alumni[0]['is_prev_alumni'])
 			{
 				$enroll_lines .= enroll_line_get($enr["user_id"], $enr["course_id"], $enr["sis_id"], $enr["section_id"], $enr["subs_id"], 'deleted').EOL;
-			}
+			}*/
```
Comment (kailasb) [inline app/modules/user_enrollment/sis.php:187]: remove commented code

**Review by kailasb — CHANGES_REQUESTED** [review]
check feedback

File: app/modules/user_enrollment/sis.php (hunk 1 above)
Comment (karuna835) [reply app/modules/user_enrollment/sis.php]: removed commented code

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**